wkcuber/export_wkw_as_tiff.py

```diff
@@ -0,0 +1,265 @@
+from argparse import ArgumentParser
+
+import logging
+import wkw
+import os
+from math import ceil
+import numpy as np
+from PIL import Image
+from typing import Tuple, Dict, Union, List
+
+from wkcuber.metadata import read_metadata_for_layer
+from wkcuber.utils import (
+    add_verbose_flag,
+    add_distribution_flags,
+    get_executor_for_args,
+    add_batch_size_flag,
+)
+from wkcuber.mag import Mag
+
+
+def create_parser():
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        "--source_path", "-s", help="Directory containing the wkw file.", required=True
+    )
+
+    parser.add_argument(
+        "--destination_path",
+        "-d",
+        help="Output directory for the generated tiff files.",
+        required=True,
+    )
+
+    parser.add_argument(
+        "--layer_name",
+        "-l",
+        help="Name of the layer that will be converted to a tiff stack",
+        default="color",
+    )
+
+    parser.add_argument("--name", "-n", help="Name of the tiffs", default="")
+
+    parser.add_argument(
+        "--bbox",
+        "-bbox",
```

Comment:
No need for a single-dash argument, since `-b` is already taken.

Reply:
As a general rule: the single-dash args should only be single character as well (e.g. `-t`). If that's not possible then leave it out.

Reply:
Thanks. I didn't know that.